resilience4j/assets/service_checks.json

```
@@ -0,0 +1,17 @@
+[
+    {
+        "agent_version": "7.59.0",
+        "integration": "Resilience4j",
+        "check": "resilience4j.prometheus.health",
+        "statuses": [
+            "ok",
+            "critical"
+        ],
+        "groups": [
+            "host",
+            "endpoint"
+        ],
+        "name": "Resilience4j endpoint health",
+        "description": "Returns `CRITICAL` if the Agent is unable to connect to the Resilience4j endpoint, otherwise returns `OK`."
+    }
+]
```

```suggestion
[]
```
This is no longer needed.